config/doctrine/User.orm.yml

```diff
@@ -153,9 +153,11 @@ App\Entity\User:
         subjectBeneficiaire:
             targetEntity: App\Entity\Beneficiaire
             mappedBy: user
+            cascade: [ "persist" ]
```

Comment:
I think that all relationships with cascade persistance or removal should be tested, as it is not really difficult and can save us from a lot of trouble